partner_default_company/models/stock_warehouse.py

```diff
@@ -0,0 +1,18 @@
+# Copyright NuoBiT Solutions, S.L. (<https://www.nuobit.com>)
```

Comment:
We don't need this file

Reply:
@MiquelRForgeFlow

Reply:
It temporal. I commented it in the `__init__.py` file. In case your solution doesn't work, we can continue to work using this file. If your solution works, then obviously I will rebase and delete it.

Reply:
Ok, perfect